File: app/src/main/java/com/brainwellnessspa/dashboardOldModule/models/PreviousAppointmentsModel.java
package com.brainwellnessspa.dashboardOldModule.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class PreviousAppointmentsModel {
    @SerializedName("ResponseData")
    @Expose
    private List<ResponseData> responseData = null;
    @SerializedName("ResponseCode")
    @Expose
    private String responseCode;
    @SerializedName("ResponseMessage")
    @Expose
    private String responseMessage;
    @SerializedName("ResponseStatus")
    @Expose
    private String responseStatus;

    public List<ResponseData> getResponseData() {
        return responseData;
    }

    public void setResponseData(List<ResponseData> responseData) {
        this.responseData = responseData;
    }

    public String getResponseCode() {
        return responseCode;
    }

    public void setResponseCode(String responseCode) {
        this.responseCode = responseCode;
    }

    public String getResponseMessage() {
        return responseMessage;
    }

    public void setResponseMessage(String responseMessage) {
        this.responseMessage = responseMessage;
    }

    public String getResponseStatus() {
        return responseStatus;
    }

    public void setResponseStatus(String responseStatus) {
        this.responseStatus = responseStatus;
    }

    public class ResponseData {
        @SerializedName("Category")
        @Expose
        private String category;
        @SerializedName("CatMenual")
        @Expose
        private String catMenual;
        @SerializedName("Image")
        @Expose
        private String image;

        public String getCategory() {
            return category;
        }

        public void setCategory(String category) {
            this.category = category;
        }

        public String getCatMenual() {
            return catMenual;
        }

        public void setCatMenual(String catMenual) {
            this.catMenual = catMenual;
        }

        public String getImage() {
            return image;
        }

        public void setImage(String image) {
            this.image = image;
        }

    }
}

File: app/src/main/java/com/brainwellnessspa/dashboardOldModule/models/PreviousAppointmentsModel.kt
package com.brainwellnessspa.dashboardOldModule.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class PreviousAppointmentsModel {
    @SerializedName("ResponseData")
    @Expose
    var responseData: List<ResponseData>? = null

    @SerializedName("ResponseCode")
    @Expose
    var responseCode: String? = null

    @SerializedName("ResponseMessage")
    @Expose
    var responseMessage: String? = null

    @SerializedName("ResponseStatus")
    @Expose
    var responseStatus: String? = null

    inner class ResponseData {
        @SerializedName("Category")
        @Expose
        var category: String? = null

        @SerializedName("CatMenual")
        @Expose
        var catMenual: String? = null

        @SerializedName("Image")
        @Expose
        var image: String? = null
    }
}